src/com/company/gamecontent/GameMapBlock.java

```diff
@@ -62,36 +68,58 @@ public GameMapBlock(int x, int y, int natType) {
                 onBuildable = true;
                 nature = Nature.SAND;
                 break;
-            case FOREST:
+            case DIRT:
+                throughWalkable = true;
+                throughShootable = true;
+                onBuildable = true;
```

Comment:
It is not possible to build on dirt.

Reply:
SAND landscape haven't building possibility too. Change it?